src/using-the-docker-image.md

```diff
@@ -1,24 +1,29 @@
-## Using the Docker image
+# Using the Docker image
 
-Currently, the recommended way to use `offchain::ipfs` is via the Docker image provided by
-[Equilibrium](https://equilibrium.co).
+The recommended way to use `offchain::ipfs` is via the [eqlabs/substrate-ipfs] image.
 
-Simply pull the [eqlabs/substrate-ipfs] image from Docker Hub:
+## Installing the image
 
 ```bash
+# Pull the image from Docker Hub
 $ docker pull eqlabs/substrate-ipfs
 ```
 
 The image comes with two binaries:
 
-1. `node-template` - which serves as a preview pallet and reference implementation
-2. `substrate-ipfs` - which is the real deal. Substrate infused with IPFS
+1. The default `node-template` serves a preview pallet and reference implementation
+2. `substrate` - the real deal, which you can develop your own pallets against
```

Comment:
help - need a better description here

Reply:
1. `node-template` contains our custom pallet to preview the Offchain::ipfs functionalities through transactions
1. `substrate` binary does not have our custom pallets to interact with the IPFS node, instead you can connect to it through its `multiaddr`

I hope this is more clear  :)

Reply:
Perfect, thank you